HULKs/hulk
# Implement interpolated parameters

## Introduced Changes

Allows to interpolate parameters in the field color based on the X-component and the orientation angle.

## ToDo / Known Issues

None

## Ideas for Next Iterations (Not This PR)

- Fancier interpolation

## How to Test

Configure different parameters and walk over the field to observe slow changes in the behavior of the robot.
